lib/Cargo.toml

```diff
@@ -15,6 +15,10 @@ hacspec-lib = { version = "0.1.0-beta.1", default-features = false, optional = t
 edhoc-crypto = { path = "../crypto", default-features = false }
 edhoc-consts = { path = "../consts", default-features = false }
 edhoc-ead = { path = "../ead", default-features = false }
+panic-semihosting = { version = "0.6.0", features = ["exit"], optional = true }
```

Comment:
Why do you need `panic-semihosting` here? Can it also be removed?

Reply:
When I try to remove it, the `staticlib` build complains that it needs a panic handler. One alternative would be just having an empty panic handler with `loop {}`, but I don't know if it can be replace by a better one later in the application, so I decided to put this in the lib.

But I think I can check about whether an application-defined panic holder would override an empty one.